app/factories/bulkrax/object_factory.rb

```diff
@@ -163,22 +163,22 @@ def create_file_set(attrs)
         uploaded_file = ::Hyrax::UploadedFile.find(uploaded_file_id)
         next if uploaded_file.file_set_uri.present?
 
-        create_file_set_actor(attrs, work, work_permissions, uploaded_file)
+        create_file_set_actor(attrs, work, work_permissions, uploaded_file, type: 'uploaded')
       end
       attrs['remote_files']&.each do |remote_file|
-        create_file_set_actor(attrs, work, work_permissions, nil, remote_file)
+        create_file_set_actor(attrs, work, work_permissions, remote_file, type: 'remote')
       end
 
       object.save!
     end
 
-    def create_file_set_actor(attrs, work, work_permissions, uploaded_file, remote_file = nil)
+    def create_file_set_actor(attrs, work, work_permissions, file, type:)
       actor = ::Hyrax::Actors::FileSetActor.new(object, @user)
-      uploaded_file&.update(file_set_uri: actor.file_set.uri)
+      file&.update(file_set_uri: actor.file_set.uri) if type = 'uploaded'
       actor.file_set.permissions_attributes = work_permissions
       actor.create_metadata(attrs)
-      actor.create_content(uploaded_file) if uploaded_file
```

Comment:
this line caused the error below because `parent` came back as `nil` in [this line ](https://github.com/samvera/hyrax/blob/92686adc219ddcf0ec9e1deece47723c4e7c7718/app/actors/hyrax/actors/file_set_actor.rb#L33)before being passed to the VisibilityCopyJob.

![Screen Shot 2022-09-06 at 6 18 50 PM](https://user-images.githubusercontent.com/29032869/188782725-760890f9-db28-40eb-b867-c6e1073fcca7.png)